app/code/Magento/InventoryInStorePickup/Model/NotifyOrderIsReadyAndShip.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model;
+
+use Magento\InventoryInStorePickupApi\Api\IsOrderReadyForPickupInterface;
+use Magento\InventoryInStorePickupApi\Api\NotifyOrderIsReadyAndShipInterface;
+use Magento\InventoryInStorePickupApi\Exception\OrderIsNotReadyForPickupException;
+
+class NotifyOrderIsReadyAndShip implements NotifyOrderIsReadyAndShipInterface
+{
+    /**
+     * @var \Magento\InventoryInStorePickupApi\Api\IsOrderReadyForPickupInterface
+     */
+    private $isOrderReadyForPickup;
+
+    /**
+     * @var \Magento\Sales\Api\ShipOrderInterface
+     */
+    private $shipOrder;
+
+    /**
+     * @var \Magento\InventoryInStorePickup\Model\Order\Email\ReadyForPickupNotifier
+     */
+    private $emailNotifier;
+
+    /**
+     * @var \Magento\Sales\Api\OrderRepositoryInterface
+     */
+    private $orderRepository;
+
+    /**
+     * NotifyOrderIsReadyAndShip constructor.
+     *
+     * @param \Magento\InventoryInStorePickupApi\Api\IsOrderReadyForPickupInterface    $isOrderReadyForPickup
+     * @param \Magento\Sales\Api\ShipOrderInterface                                    $shipOrder
+     * @param \Magento\InventoryInStorePickup\Model\Order\Email\ReadyForPickupNotifier $emailNotifier
+     * @param \Magento\Sales\Api\OrderRepositoryInterface                              $orderRepository
```

Comment:
why do you use this kind of formatting?
what would happen if you need to add one more parameter with a class name longer than
`\Magento\InventoryInStorePickup\Model\Order\Email\ReadyForPickupNotifier`
?
whether you adjust the indentation for all existing parameters?

Reply:
We use such code style in the ISM community just because we can (All the settings are synchronized) and just because it looks more readable, as for me.

Will be fixed to be up to date with M2.

Reply:
Probably fixed